src/locales/ja-JP/admin.ftl

```diff
@@ -0,0 +1,1537 @@
+configure-webhooks-rotateSigningSecretHelper =
+  有効期限が切れると、古いシークレットで署名が生成されなくなります。
+configure-webhooks-rotateSigningSecretSuccessUseNewSecret =
+  Webhookエンドポイント署名シークレットがローテーションされました。
+ 以下の新しいシークレットを使用するように統合を更新してください。
+configure-webhooks-disableEndpoint = エンドポイントを無効にする
+configure-webhooks-disableEndpointDescription =
+  このエンドポイントは現在有効です。
+このエンドポイントを無効にすると、指定されたURLに新しいイベントが送信されなくなります。
+configure-webhooks-disableEndpointButton = エンドポイントを無効にする
+configure-webhooks-enableEndpoint = エンドポイントを有効にする
+configure-webhooks-enableEndpointDescription =
+  このエンドポイントは現在無効になっています。
+ このエンドポイントを有効にすると、指定されたURLに新しいイベントが送信されます。
+configure-webhooks-enableEndpointButton = エンドポイントを有効にする
+configure-webhooks-deleteEndpoint = エンドポイントを削除する
+configure-webhooks-deleteEndpointDescription =
+  エンドポイントを削除すると、指定されたURLに新しいイベントが送信されなくなります。
+configure-webhooks-deleteEndpointButton = エンドポイントを削除する
+configure-webhooks-endpointStatus = エンドポイントの状態
+configure-webhooks-signingSecret = 署名シークレット
+configure-webhooks-signingSecretDescription =
+  次の署名シークレットは、URLに送信される要求ペイロードに署名するために使用されます。
+ Webhook署名の詳細については、<externalLink> Webhookガイド</ externalLink>にアクセスしてください。
+configure-webhooks-expiresOldSecret = 古い署名シークレットを期限切れにする
+configure-webhooks-expiresOldSecretImmediately = すぐに
+configure-webhooks-expiresOldSecretHoursFromNow =
+  { $hours ->
+    [1] 1時間後
+    *[other] { $hours } 時間後
+  }  今から
+configure-webhooks-detailsButton = 詳細を確認するには<icon>キーボードの右矢印</icon>
+
+### General
+configure-general-guidelines-title = コミュニティガイドラインの概要
+configure-general-guidelines-explanation =
+  これは、サイト全体のコメントの上に表示されます。
+ Markdownを使用してテキストをフォーマットできます。
+   Markdownの使用方法の詳細はこちら<externalLink>https://www.markdownguide.org/cheat-sheet/</externalLink>
+configure-general-guidelines-showCommunityGuidelines = コミュニティガイドラインの概要を表示する
+
+#### Bio
+configure-general-memberBio-title = メンバーの経歴
+configure-general-memberBio-explanation =
+  コメント投稿者がプロフィールに略歴を追加できるようにします。
+ 注：これにより、メンバーの経歴を報告できるため、モデレーターの作業負荷が増加する可能性があります。
+configure-general-memberBio-label = メンバーの経歴を許可する
+
+#### Locale
+configure-general-locale-language = 言語
+configure-general-locale-chooseLanguage = Coralコミュニティの言語を選択してください。
+configure-general-locale-invalidLanguage =
+  以前お選びいただいた言語<lang> </ lang>は、現在は存在しません。 別の言語を選択してください。
+
+#### Sitewide Commenting
+configure-general-sitewideCommenting-title = サイト全体のコメント
+configure-general-sitewideCommenting-explanation =
+  サイト全体の新しいコメントのコメントストリームを開くか閉じます。
+   新しいコメントがオフになっている場合、新しいコメントを送信することはできません。
+ ただし、既存のコメントは引き続き反応を受け取り、報告され、共有される可能性があります。
+configure-general-sitewideCommenting-enableNewCommentsSitewide =
+  サイト全体で新しいコメントを有効にする
+configure-general-sitewideCommenting-onCommentStreamsOpened =
+  オン-新しいコメントのために開かれたコメントストリーム
+configure-general-sitewideCommenting-offCommentStreamsClosed =
+  オフ-新しいコメントのためにコメントストリームを閉じました
+configure-general-sitewideCommenting-message = サイト全体のクローズドコメントメッセージ
+configure-general-sitewideCommenting-messageExplanation =
+  コメントストリームがサイト全体で閉じられたときに表示されるメッセージを記述します
+
+#### Embed Links

```

Comment:
It looks like the rest of this file is in English. Did you mean to include more Japanese translations here?